OWNERS_ALIASES

```diff
@@ -1,19 +1,21 @@
 # See the OWNERS docs: https://www.kubernetes.dev/docs/guide/owners
-# This file should be kept in sync with k/org.
 
 aliases:
-  # Reference: https://github.com/kubernetes/org/blob/main/OWNERS_ALIASES
   sig-network-leads:
-    - aojea
-    - danwinship
-    - mikezappa87
-    - shaneutt
-    - thockin
+  - aojea
+  - danwinship
+  - mikezappa87
+  - shaneutt
+  - thockin
 
   ingate-maintainers:
-    - cpanato
-    - Gacko
-    - puerco
-    - rikatz
```

Comment:
Ricardo wants to work on ingate, so let's leave in here.

Reply:
Ah, ok. I removed him because he removed himself from InGate related teams/owners elsewhere.

Reply:
https://github.com/kubernetes/org/pull/5305